plugins/sender/plugin.go

```diff
@@ -18,7 +18,11 @@ func NewRemote(endpoint string) model.SenderService {
 
 func (p *plugin) SenderAllowed(user *model.User, repo *model.Repo, build *model.Build, conf *model.Config) (bool, error) {
 	path := fmt.Sprintf("%s/senders/%s/%s/%s/verify", p.endpoint, repo.Owner, repo.Name, build.Sender)
-	err := internal.Send("POST", path, build, nil)
+	data := map[string]interface{}{
+		"build": build,
+		"config": conf,
+	}
+	err := internal.Send("POST", path, data, nil)
 	if err != nil {
```

Comment:
I think it needs to be `&data` (pointer) or the json package will return a marshal error. Sorry I didn't mention this previously